jupyterhub/tests/test_spawner.py

```diff
@@ -483,3 +483,49 @@ def options_from_form(form_data, spawner):
     for key, value in form_data.items():
         assert key in result
         assert result[key] == value
+
+
+def test_spawner_server(db):
+    spawner = new_spawner(db)
+    spawner.orm_spawner = None
+    orm_spawner = orm.Spawner()
+    orm_server = orm.Server(base_url="/1/")
+    orm_spawner.server = orm_server
+    db.add(orm_spawner)
+    db.add(orm_server)
+    db.commit()
+    # initial: no orm_spawner
+    assert spawner.server is None
+    # assign orm_spawner sets server
```

Comment:
```suggestion
    # assert orm_spawner sets server
```

Reply:
I actually meant `assign` since it's assigning the spawner.orm_spawner attribute that updates the server value. I've updated the comments to hopefully be clearer.

Reply:
Oh, right! sorry for getting it wrong and thanks for updating the comments